internal/cloud/realm/client.go

```diff
@@ -35,6 +38,7 @@ type Client interface {
 	DeployDraft(groupID, appID, draftID string) (AppDeployment, error)
 	DiffDraft(groupID, appID, draftID string) (AppDraftDiff, error)
 	DiscardDraft(groupID, appID, draftID string) error
+	Deployments(groupID, appID string) ([]AppDeployment, error)
```

Comment:
⚠️ added this method solely for the purposes of testing

In order to wait for the `DEPLOY_DEPENDENCIES` job to complete, I needed to get all deployments to find the right `ID`